netadapter/server/grpcserver/protowire/wire.go

```diff
@@ -0,0 +1,38 @@
+package protowire
+
+import (
+	"bytes"
+
+	"github.com/kaspanet/kaspad/wire"
+)
+
+// ToWireMessage converts a KaspadMessage to it's wire.Message representation
+func (x *KaspadMessage) ToWireMessage() (wire.Message, error) {
```

Comment:
Why the receiver is called `x`?

Reply:
KaspadMessage is auto-generated by protobuf, and protobuf calls the receiver x